src/apps/vpn/platforms/android/androidutils.cpp

```diff
@@ -29,7 +29,35 @@ constexpr auto UTILS_CLASS = "org/mozilla/firefox/vpn/qt/VPNUtils";
 }  // namespace
 
 // static
-QString AndroidUtils::GetDeviceName() {
+QString AndroidUtils::getDeviceName() {
```

Comment:
Isn't this new function the same as AndroidUtils::getDeviceModel?

Reply:
Thanks! Splitted them up